src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const spans = document.querySelectorAll('.population');
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+
+const numbers = Array.from(spans)
+  .map((span) => span.textContent.trim().replace(/,/g, ''))
+  .filter((text) => !isNaN(text) && text !== '')
+  .map(Number);
+
+const total = numbers.reduce((sum, num) => sum + num, 0);
+const average = Math.round(total / numbers.length);
```

Comment:
There is a potential issue here if the numbers array is empty, as dividing by zero will result in NaN. Consider adding a check to ensure numbers.length is not zero before performing the division.